graftm/db_search_results.py

```diff
@@ -0,0 +1,48 @@
+from graftm.deduplicator import Deduplicator
+from graftm.sequence_io import SequenceIO
+
+import logging
+import os
+
+class DBSearchResult:
+    '''Class for containing results from search pipeline in GraftM '''
+    def __init__(self, ouput_reads, search_result, hit_read_count, slash_endings):
```

Comment:
some more comments for this class would be good I think, given it is a new class